src/SFA.DAS.ApprenticeCommitments.Web/Services/OuterApi/IOuterApiClient.cs

```diff
@@ -28,5 +28,10 @@ public interface IOuterApiClient
             [Path] Guid apprenticeid, [Path] long apprenticeshipid,
             [Body] EmployerConfirmationRequest confirmation);
 
+        [Post("/apprentices/{apprenticeid}/apprenticeships/{apprenticeshipid}/apprenticeshipdetailsconfirmation")]
+        Task ConfirmApprenticeshipDetails(
+            [Path] Guid apprenticeid, [Path] long apprenticeshipid,
+            [Body] ApprenticeshipDetailsRequest confirmation);

```

Comment:
Minor suggestion - we are confirming the apprenticeship details with this post aren't we?

```suggestion
            [Body] ApprenticeshipDetailsConfirmationRequest confirmation);

```

Reply:
Agree, I've amended the file name